src/functions-and-arrays.js

```diff
@@ -1,41 +1,109 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1,num2) {
+  return num1>=num2?num1:num2;
+}
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(arr) {
+  let aux = "";
+  let count = 0;
+  arr.forEach(element => {
+    if(element.length > aux.length){
+      aux = element;
+    } 
+    count++;
+  });
+  return !count?null:aux;
```

Comment:
Aqui eu recomendaria fazer o return assim:
```javascript
return !arr.length ? null : aux;
```
assim nem precisa criar outra variável.
porém por se tratar de uma validação de dados de entrada (que se não for o esperado, pode quebrar seu código) normalmente colocamos a validação antes de tudo na função.